config/gitleaks.toml

```diff
@@ -489,9 +489,6 @@ keywords = [
     "key","api","token","secret","client","passwd","password","auth","access",
 ]
 [rules.allowlist]
-paths = [
```

Comment:
@icecream-monster @JoostVoskuil noticed that this gets removed when generating a new config. I'll have to update the template and contributing guidelines so that `allowlists` can be respected. In the mean time, I'll merge this as it LGTM and open a follow up PR that re-introduces the `Database.refactorlog` allowlist (or if you're willing to make the change in this MR @icecream-monster, that would be appreciated too)

Reply:
sorry this was missed, I pushed new commit to the PR ensuring `Database.refactorlog` part remains untouched